driver-core/src/test/java/com/datastax/driver/core/ProtocolV1Test.java

```diff
@@ -24,7 +25,7 @@
 /**
  * Tests targeting protocol v1 specifically.
  */
-@CCMConfig(version = "1.2.19")
+@CassandraVersion(major = 1.2, exactRequired = true)
```

Comment:
So if I get it right, this test used to always be run against a 1.2.19 cluster, regardless of the default CCM version (`cassandra.version`), whereas now it would only run if the default CCM version _is_ 1.2.X, and would be skipped for all other configurations.

I'm fine with that, but indeed my feeling is that we should avoid skipping tests. IMHO there should be a default CCM version for the test suite that would be used for all tests that do not declare any specific CCM requirement, but if a test – like this one – has very specific requirements, such as a specific version or a specific version range, then a non-default CCM cluster should be spun for it, rather than simply skipping it. I basically came to this conclusion seeing how many times we realize very late that a test fails for a specific C* version, just because the test was being skipped on our local machines.

In the new toolbox repository, things are being done that way (there is no test skipping). I understand that in a Jenkins matrix build that would make some tests be executed several times always against the same C* version, but I'd rather have that, than risking to have some tests maybe being skipped everywhere.